cmd/dfdaemon/main.go

```diff
@@ -17,9 +17,27 @@
 package main
 
 import (
+	"math/rand"
+	"os"
+	"time"
+
 	"github.com/dragonflyoss/Dragonfly/cmd/dfdaemon/app"
+	dferr "github.com/dragonflyoss/Dragonfly/pkg/errortypes"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 )
 
 func main() {
-	app.Execute()
+	rand.Seed(time.Now().UnixNano())
```

Comment:
What is the meaning of this line of code?

Reply:
See https://golang.org/pkg/math/rand/#Rand.Seed.